test/TestExpectations.json

```diff
@@ -299,6 +299,13 @@
     "expectations": ["SKIP"],
     "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
   },
+  {
+    "testIdPattern": "[dialog.spec] Page.Events.Dialog *",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["firefox", "webDriverBiDi"],
+    "expectations": ["SKIP"],
```

Comment:
Was there actually a reason why these were marked as `SKIP` and not `FAIL`? I'll provide a PR in a moment which will remove this entry again.

Reply:
Thanks for flagging. @Lightning00Blade could you please check if we can work around by not setting the capability for Firefox yet and roll back the test expectation changes?

Reply:
You can leave it as is. I've a fix which landed and will be in the next Nightly build that is going to be available in the next 2-3h.